.github/workflows/lint.yml

```diff
@@ -0,0 +1,35 @@
+---
+name: Lint
+
+on:
+  pull_request:
+
+jobs:
+  super-lint:
+    name: Lint with Super-Linter
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks out a copy of your repository on the ubuntu-latest machine
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          # Make sure the actual branch is checked out when running on PR
+          ref: ${{ github.event.pull_request.head.sha }}
+          # Full git history needed to get proper list of changed files
+          fetch-depth: 0
+
+      # Runs the Super-Linter action
+      - name: Run Super-Linter on new changes
+        uses: docker://ghcr.io/github/super-linter:slim-v4
+        env:
+          DEFAULT_BRANCH: main
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          MARKDOWN_CONFIG_FILE: .markdownlint.json
+          # Only check new or edited files
+          VALIDATE_ALL_CODEBASE: false
+          # Fail on errors
+          DISABLE_ERRORS: false
+          # shellcheck conf
+          SHELLCHECK_OPTS:
+            -e SC1090,SC2046,SC2164,SC2166,SC1091,SC2086,SC2001,SC2219,SC2181
```

Comment:
So i think this list is inherited from here I think: https://github.com/EGI-Federation/bdii/pull/35
I don't remember the exact reasons for them, but it can certainly be adapted.
Have you any suggestion?

Reply:
I opened #5 for checking this in the future.

Reply:
Great, thanks!